drape_frontend/drape_engine.cpp

```diff
@@ -477,4 +476,11 @@ void DrapeEngine::SetKineticScrollEnabled(bool enabled)
                                   MessagePriority::High);
 }
 
+void DrapeEngine::SetTimeInBackground(double time)
+{
+  m_threadCommutator->PostMessage(ThreadsCommutator::RenderThread,
+                                  make_unique_dp<SetTimeInBackgroundMessage>(time),
```

Comment:
Не логичнее ли тогда пробрасывать уход в фон и восстановление в drape?

> On 14.4.2016, at 12.34, Roman Kuznetsov <notifications@github.com> wrote:
> 
> In drape_frontend/drape_engine.cpp:
> 
> > @@ -477,4 +476,11 @@ void DrapeEngine::SetKineticScrollEnabled(bool enabled)
> >                                    MessagePriority::High);
> >  }
> >  
> > +void DrapeEngine::SetTimeInBackground(double time)
> > +{
> > +  m_threadCommutator->PostMessage(ThreadsCommutator::RenderThread,
> > +                                  make_unique_dp<SetTimeInBackgroundMessage>(time),
> Нет, нужна дельта по времени, которую мы провели в фоне или выключенном состоянии
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>